rs/tests/ict/cmd/helpers.go

```diff
@@ -149,3 +151,101 @@ func get_closest_target_matches(all_targets []string, target string) []string {
 		return len(s) > 0
 	})
 }
+
+func sparse_checkout(repoUrl, repoDir string, sparseCheckoutPaths []string) (string, error) {
+	startingPoint, err := os.Getwd()
+	if err != nil {
+		return "", fmt.Errorf("Could not get current dir: %v", err)
+	}
+
+	if repoDir == "" {
+		homedir, err := os.UserHomeDir()
+		if err != nil {
+			return "", fmt.Errorf("Failed to get home dir: %v", err)
+		}
+		repoDir = filepath.Join(homedir, ".cache", "k8s_repo")
```

Comment:
would make sense to use [XDG variables](https://specifications.freedesktop.org/basedir-spec/latest/) if available

Reply:
Had a quick read [here](https://pkg.go.dev/os#UserHomeDir). Since that is how I fetch homeDir I believe its XDG compliant.